modules/ROOT/nav.adoc

```diff
@@ -1,16 +1,27 @@
 .xref:index.adoc[Anypoint Partner Manager]
 * xref:index.adoc[Overview]
```

Comment:
INSTRUCTIONS FOR EDIT

Hi @kim.bostrom, as promised, here are the files for the new features in PM 2.0.  Just the green parts and I hope to get this back by the end of the day on Thursday. I will have the RNs for you Friday am.